docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -547,6 +547,21 @@ You can use the [`moved-lab-modules` codemod](https://github.com/mui-org/materia
   +<BottomNavigation onChange={(event: React.SyntheticEvent) => {}} />
   ```
 
+### BottomNavigationAction
+
+- `span` element that wraps children has been removed. `wrapper` classKey is also removed. More details about [this change](https://github.com/mui-org/material-ui/pull/26666).
```

Comment:
```suggestion
- Remove the `span` element that wraps the children. Remove the `wrapper` classKey too. More details about [this change](https://github.com/mui-org/material-ui/pull/26666).
```

Reply:
No objections to change the approach, one header by React component instead of UI component, but maybe the change should be done all at once, in a batch 🤷‍♂️